contributors/guide/README.md

```diff
@@ -0,0 +1,212 @@
+* Ultimately, all contributor guide specific files will live in this folder.
+
+Please find _Improvements needed_ sections below and help us out.
+
+For example:
+
+_Improvements needed_
+* kubernetes/kubernetes/contributing.md -> point to this guide
+
+* kubernetes/community/CONTRIBUTING.md -> Needs a rewrite 
+
+* kubernetes/community/README.md -> Needs a rewrite 
+
+* Individual SIG contributing documents -> add a link to this guide
+
+* Generate ToC for the section below 
+
+# Welcome
+
+Welcome to Kubernetes! This document is the single source of truth for how to contribute to the code base. Please leave comments / suggestions if you find something is missing or incorrect.
+
+# Before you get started
+
+## Sign the CLA
+
+Before you can contribute, you will need to sign the [Contributor License Agreement](https://github.com/kubernetes/community/blob/master/CLA.md).
+
+## Setting up your development environment
+
+If you haven’t set up your environment, please find resources [here](https://github.com/kubernetes/community/tree/master/contributors/devel). These resources are not well organized currently; please have patience as we are working on it.
+
+_Improvements needed_
+* A new developer guide will be created and linked to in this section.
+
+    * RyanJ from Red Hat is working on this
+
+## Community Expectations 
+
+Kubernetes is a community project. Consequently, it is wholly dependent on its community to provide a productive, friendly and collaborative environment.
+
+The first and foremost goal of the Kubernetes community to develop orchestration technology that radically simplifies the process of creating reliable distributed systems. However a second, equally important goal is the creation of a community that fosters easy, agile development of such orchestration systems.
+
+We therefore describe the expectations for members of the Kubernetes community. This document is intended to be a living one that evolves as the community evolves via the same PR and code review process that shapes the rest of the project. It currently covers the expectations of conduct that govern all members of the community as well as the expectations around code review that govern all active contributors to Kubernetes.
+
+### Code of Conduct
+
+Please make sure to read and observe our [Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)
+
+### Code Review
+
+For a brief description of the importance of code review, please read [On Code Review](https://github.com/kubernetes/community/blob/master/contributors/devel/community-expectations.md#code-review).
+
+_Improvements needed_
+* edit above link to only provide code review section.
+* decide whether this is something that belongs in this section, or should be combined with Contributing instructions [below](https://github.com/guineveresaenger/community/tree/contributor-guide/contributors/guide#contributing).
```

Comment:
Links to a personal repo, per above, this should be a relative link (e.g. `/contributor-guide/contributors/guide#contributing`)

Reply:
whoops